### PR TITLE
Implement Linking/Unlinking an Existing User to Organization SSO

### PR DESCRIPTION
**Pull request by addisonbeck**
### Objective
Create a server environment that can send the necessary information for linking an existing user to an org sso, as well as unlinking

### Code changes

- Created an API call and repository method for deleting dbo.SsoUser records that matches the already existing stored procedure

- Added org Identifier to some organization response models and views so we can skip that step when we already know what organization a user is authenticating for, like with this intended linking feature

### CODE REVIEW

File: src/Api/Controllers/SsoUserController.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        private readonly ISsoUserRepository _ssoUserRepository;
+
+        public SsoUserController(IUserService userService, ISsoUserRepository ssoUserRepository)
+        {
+            _userService = userService;
+            _ssoUserRepository = ssoUserRepository;
+        }
+
+        [HttpDelete("{organizationId}")]
+        public async Task Delete(string organizationId)
```
Comment (cscharf) [inline src/Api/Controllers/SsoUserController.cs:24]: let's add this to the existing `AccountsController` as `DeleteSsoUser` or something to that tune and remove the new `SsoUserController`.

Comment (kspearrin) [reply src/Api/Controllers/SsoUserController.cs:24]: Yes, lest just create `DELETE /accounts/sso/orgId` or something

File: src/Api/Controllers/SsoUserController.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            if (!userId.HasValue) {
+                throw new NotFoundException();
+            }
+
+            var ssoUser = new SsoUser() {
+                UserId = userId.Value,
+                OrganizationId = new Guid(organizationId),
+            };
+
+            await _ssoUserRepository.DeleteAsync(ssoUser);
```
Comment (kspearrin) [inline src/Api/Controllers/SsoUserController.cs:36]: I wonder if we should track an event log for this?

Comment (kspearrin) [reply src/Api/Controllers/SsoUserController.cs:36]: probably fine for now

**Pull request closed without merging**